recipe/build.sh

```diff
@@ -3,6 +3,9 @@
 if [ "$(uname)" == "Darwin" ]; then
     export CXX="${CXX} -stdlib=libc++"
     export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
+else
+    # Shared Fortran libs on OSX unsupported as of 1.8.18
+    FORTRAN_FLAGS=("--enable-fortran" "--enable-fortran2003")
```

Comment:
@jakirkham While digging into why the Fortran tests were failing on OSX, I came across a log message saying the following:

    checking if shared Fortran libraries are supported... no
    configure: WARNING: Shared Fortran libraries not currently supported on Mac.
    configure: WARNING: Allowing unsupported Fortran shared libraries due to use of --enable-unsupported flag

The ```--enable-unsupported``` flag is needed to build the high-level API and enable thread-safe (which is *not* supported by the library authors), so we can't turn that off.